### DISCUSSION TITLE
Problems with Visualizing Model Outputs

### DISCUSSION THREAD

**Discussion by DEQ21** [talmolab/sleap#907] [Help!]
I recently have started the process of labeling and using SLEAP to cover most of the labels in the video. However, I am trying to generate the confidence graphs and velocity graphs that appear in the validation step. When I press visualize model outputs I get this : "TypeError: Can't instantiate abstract class VisualPredictor with abstract methods _initialize_inference_model, is_grayscale"
or I get a heading error. Could someone let me know how I should proceed?

**Comment by talmo** [comment]
Hi @DEQ21,

The error is definitely a bug and we'll be tracking it in a new issue (@roomrys).

In the meantime, here's a minimal workaround to view model inference results: https://colab.research.google.com/drive/1qac6PdLSssYdYXFHz5lrsrUPKUmRTcPN?usp=sharing

This is basically what you'd get with the VisualPredictor.

I'm not sure what you mean by velocity graphs though. During training you're shown the loss curves, and those are saved in the `loss_history.csv` file in the model folder if you want to replot it.

If you want velocities of the landmark movements, check out the [analysis examples notebook](https://sleap.ai/notebooks/Analysis_examples.html).

Let us know if you have any questions!

Cheers,

Talmo

**Reply by roomrys** [reply]
Issue created, see #908 for updates.
